project/plugins.sbt

```diff
@@ -11,7 +11,7 @@ addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "0.9.3")
 addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "0.1.18")
 addSbtPlugin("org.scalariform" % "sbt-scalariform" % "1.8.1")
 addSbtPlugin("com.dwijnand" % "sbt-dynver" % "2.0.0")
-addSbtPlugin("com.typesafe.sbt" % "sbt-osgi" % "0.9.2")
+// addSbtPlugin("com.typesafe.sbt" % "sbt-osgi" % "0.9.2")
```

Comment:
Perhaps remove it entirely instead of commenting it out?

Reply:
Sure, perhaps is better that way. I left it there hoping that one day, I'll resurrect the OSGi support.

Reply:
It'll be easy to add it back ;)